pkg/rke/templates/templates.go

```diff
@@ -252,10 +254,18 @@ func LoadK8sVersionedTemplates() map[string]map[string]string {
 			">=1.25.9-rancher2-1 <1.25.9-rancher2-2":   aciv5236,
 			">=1.25.9-rancher2-2 <1.25.13-rancher1-1":  aciv5271,
 			">=1.25.13-rancher1-1 <1.25.16-rancher2-2": aciv6031,
-			">=1.25.16-rancher2-2 <1.26.0-rancher0":    aciv6032,
+			">=1.25.16-rancher2-2 <1.25.16-rancher2-3": aciv6032,
+			">=1.25.16-rancher2-3 <1.26.0-rancher0":    aciv6033,
 			">=1.26.0-rancher0 <1.26.8-rancher1-1":     aciv5271,
 			">=1.26.8-rancher1-1 <1.26.11-rancher2-2":  aciv6031,
-			">=1.26.11-rancher2-2":                     aciv6032,
+			">=1.26.11-rancher2-2 <1.26.11-rancher2-3": aciv6032,
+			">=1.26.11-rancher2-3 <1.26.14-rancher1-1": aciv6033,
```

Comment:
We do have an existing version `v1.26.13-rancher1-1` that references ACI 6.0.3.2 images and this new range will cover it and result in 6.0.3.3 template used for it.
So we either introduce `v1.26.13-rancher1-2` instead of `v1.26.11-rancher2-3` or further split ranges to ensure `v1.26.13-rancher1-1` continues to use `aciv6032` template.

Reply:
updated the version and references for 6033